daemon/mgr/container_prune.go

```diff
@@ -0,0 +1,109 @@
+package mgr
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/alibaba/pouch/apis/types"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Prune implement interface
+func (mgr *ContainerManager) Prune(ctx context.Context, option *ContainerListOption) (*types.ContainerPruneResp, error) {
+	var rep types.ContainerPruneResp
+
+	containerLists, err := mgr.Store.List()
+	if err != nil {
+		return nil, err
+	}
+
+	fc, err := newFilterContext(option)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, obj := range containerLists {
+		c, ok := obj.(*Container)
+		if !ok {
+			return nil, fmt.Errorf("failed to get container list, invalid meta type")
+		}
+
+		if !c.IsRunning() {
+			// By default, all are deleted. With matching conditions, only the matching conditions are deleted.
+			if len(option.Filter) > 0 && false == fc.filterStoppedContainer(c) {
+				continue
+			}
+
+			if c.SizeRw > 0 {
```

Comment:
Here, I take `c.SizeRw` as reclaimed space. is it right?

Reply:
I check the logic in moby, reclaimedSpace collect the size of RW layer of container, I think it's right. But it's better wating someone to double check this.